doc/api/worker_threads.md

```diff
@@ -47,8 +47,8 @@ if (isMainThread) {
 }
 ```
 
-The above example spawns a Worker thread for each `parse()` call. In actual
-practice, use a pool of Workers for these kinds of tasks. Otherwise, the
+The above example spawns a Worker thread for each `parseJSAsync()` call. In
+actual practice, use a pool of Workers for these kinds of tasks. Otherwise, the
```

Comment:
We can probably remove `actual` here (I think that's what Colin was suggesting)
```suggestion
practice, use a pool of Workers for these kinds of tasks. Otherwise, the
```